src/app/drivingLicence/fields.js

```diff
@@ -47,7 +53,11 @@ module.exports = {
     validate: [
       "required",
       "date",
-      { type: "before", arguments: [new Date().toISOString().split("T")[0]] }
+      { type: "before", arguments: [new Date().toISOString().split("T")[0]] },
+      {
+        type: "dvlaChecker",
+        ...dvlaValidatorObj,
+      }
```

Comment:
Had a thought, we don't want this validation if it's a dva screen, for now lets take the validation off of the dob field and leave it in for the licence number, and we can always circle back once emily has confirmed the validation msg/wether or not we want it on both fields or just one